02_bubble/joshcarp/bubble.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println(bubble([]int{3, 2, 1, 5}))
+	fmt.Println(insertion([]int{3, 2, 1, 5}))
+}
+
+func bubble(s []int) []int {
+	n := len(s)
+	for i := 0; i < n; i++ {
+		for j := 0; j < n-i-1; j++ {
+			if s[j] > s[j+1] {
+				s[j], s[j+1] = s[j+1], s[j]
+			}
+		}
+	}
+	return s
```

Comment:
The requirement says "returns a sorted `copy` of int slice s". My interpretation is that it does not touch the input s but return a `copy` which is sorted. So returning `s` here seems not doing that to me.
Same as insertion